public/app/plugins/datasource/tempo/configuration/QuerySettings.tsx

```diff
@@ -0,0 +1,72 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { DataSourcePluginOptionsEditorProps, updateDatasourcePluginJsonDataOption } from '@grafana/data';
+import { InlineField, InlineFieldRow, Input, useStyles } from '@grafana/ui';
+
+import { TempoJsonData } from '../types';
+
+interface Props extends DataSourcePluginOptionsEditorProps<TempoJsonData> {}
+
+export function QuerySettings({ options, onOptionsChange }: Props) {
+  const styles = useStyles(getStyles);
+
+  return (
+    <div className={styles.container}>
+      <h3 className="page-heading">TraceID Query</h3>
+      <InlineFieldRow>
+        <InlineField
+          label="Span start time shift"
+          labelWidth={26}
+          grow
+          tooltip="Shifts the start time of the span. Default 0 (Time units can be used here, for example: 5s, 1m, 3h)"
```

Comment:
I think this needs a bit longer explanation and better name.

label: Time shift for start of search time range
tooltip: Shifts the start of the time range when searching by trace ID. This is needed as searching for traces can return traces that do not fully fall into the search time range. If trace is searched by trace id in the same time range it won't show up. This shift should allow for traces to fall fully into the shifter time range. The longer your traces the higher the time shift should be used. Default 30m (Time units can be used here, for example: 5s, 1m, 3h).

@joe-elliott does this make sense? Also I wonder do we need start shift? Don't we only need the end shift? Meaning does the search return traces that started before the search time range but overlap a bit? Cause I would assume we need only end range shift.